### PR TITLE
Feat- signup 

### PR DESCRIPTION
**Pull request by surbhi84**
Live link
https://hallodecormockbee.netlify.app/

### CODE REVIEW

File: src/components/auth/Signup.jsx [hunk 1]
```diff
@@ -37,17 +99,46 @@ export function Signup() {
...
+          <button
+            className="cart-btn full-width"
+            onClick={async () => {
+              console.log(
+                signupInfo.firstName,
+                signupInfo.lastName,
+                signupInfo.email,
+                signupInfo.password,
+                signupInfo.confirmPwd,
+                signupInfo.tAc
```
Comment (anupkgurung) [inline src/components/auth/Signup.jsx:120]: you can use console.table(signupInfo) instead would save you the hustle of debugging.

**Review by anupkgurung — APPROVED** [review]
seems good to me.
api js seems to be a custom hook you could relocate it later to some specified folder

File: src/components/auth/Signup.jsx (hunk 1 above)
Comment (surbhi84) [reply src/components/auth/Signup.jsx:120]: I forgot to remove that after debugging thanks for pointing out.
also api.js is not a custom hooks file, it has functions regarding api calls.

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const BASEURL = "";
```
Comment (viru2001) [inline src/api.js:3]: you are not using this variable so you can remove it

File: src/components/auth/Signup.jsx [hunk 3]
```diff
@@ -10,25 +44,53 @@ export function Signup() {
         >
           <h3 className={auth["auth-heading"]}>Sign up</h3>
 
-          {["First Name", "Last Name", "Email id"].map((label) => (
+          {[
+            { label: "First Name", type: "FIRSTNAME" },
+            { label: "Last Name", type: "LASTNAME" },
+            { label: "Email id", type: "EMAIL" },
+          ].map(({ label, type }) => (
```
Comment (viru2001) [inline src/components/auth/Signup.jsx:51]: I liked this. but you can store this array in other file

**Review by viru2001 — COMMENTED** [review]
Everything looks good, gave small suggestions

**Pull request merged**